Interop/ModelCard/src/test/java/ExternalModelsTest.java

```diff
@@ -0,0 +1,49 @@
+import org.junit.jupiter.api.Test;
+import org.tribuo.Model;
+import org.tribuo.MutableDataset;
+import org.tribuo.classification.Label;
+import org.tribuo.classification.LabelFactory;
+import org.tribuo.classification.xgboost.XGBoostClassificationConverter;
+import org.tribuo.common.xgboost.XGBoostExternalModel;
+import org.tribuo.datasource.IDXDataSource;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class ExternalModelsTest {
+    @Test
+    public void testExternalModelException() throws IOException {
+        var labelFactory = new LabelFactory();
+        var mnistTrainSource = new IDXDataSource<>(Paths.get("src/test/input-data/train-images-idx3-ubyte.gz"),Paths.get("src/test/input-data/train-labels-idx1-ubyte.gz"),labelFactory);
```

Comment:
We can't ship MNIST, so instead just fake up some dataset that has 784 features and 10 labels. As the main result of this test is to throw an exception it doesn't matter if the evaluation is nonsensical.

Reply:
I've now created a my own mock data as suggested.